saleor/dashboard/seo/utils.py

```diff
@@ -0,0 +1,41 @@
+from django.utils.translation import pgettext_lazy
+
+from ...core.utils.text import strip_html_and_truncate
+from ..widgets import CharsLeftWidget
+
+MIN_TITLE_LENGTH = 25
+MIN_DESCRIPTION_LENGTH = 120
+SEO_HELP_TEXTS = {
+    'seo_description': pgettext_lazy(
+        'Form field help text',
+        (
+            'If empty, the preview shows what will be autogenerated.')),
+    'seo_title': pgettext_lazy(
+        'Form field help text',
+        (
+            'If empty, the preview shows what will be autogenerated.'))}
+SEO_LABELS = {
+    'seo_description': pgettext_lazy(
+        (
+            'Field name, Meta Description is page summary '
+            'used by Search Engines'),
+        'Meta Description'),
```

Comment:
I just pulled the merged changes... There is a "SEO Title" label and a "Meta Description" label. I think we should choose between Meta and SEO, and not use both. I would go for "SEO Description" instead of "Meta".

Reply:
Meta description is more precise https://moz.com/learn/seo/meta-description
Title has SEO prefix to not confuse with regular `Title` eg. on page model

Reply:
It's not a big issue but I think it's quite reasonable to have those fields named in a similar manner.

Reply:
My pet peeve is that most fields use "Sentence case" for their labels yet these two use "Title Case" so they stand out. Could we rename them "Search engine title" and "Search engine summary" or something similar?